include/net/lwm2m_client_utils.h

```diff
@@ -120,6 +120,20 @@ bool lwm2m_security_needs_bootstrap(void);
  * @brief Initialize Device object
  */
 int lwm2m_init_device(void);
+
+/**
+ * @brief Reboot handler for a device object
+ *
+ * All arguments are ignored.
+ *
+ * @param obj_inst_id Device object instance.
+ * @param args Argument pointer's
+ * @param args_len Length of argument's
+ *
+ * @return 0 Reboot trigged.
+ * @return < 0 Reboot failured.
```

Comment:
There should only be a single `@return` entry in the API docs, if you want to list mutliple values use `@retval` tag instead.